network-api/networkapi/wagtailpages/factory/bannered_campaign_page.py

```diff
@@ -45,9 +48,23 @@ def generate(seed):
     reseed(seed)
 
     print('Generating Bannered Campaign Pages under namespace')
-    BanneredCampaignPageFactory.create(
-        parent=bannered_campaign_namespace,
-        title="Test Bannered Campaign"
-    )
+    title = 'Test Bannered Campaign'
```

Comment:
Can we change the title to include "fixed title" like what we did for [fake blog post](https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/wagtailpages/factory/blog.py#L80)? For people who are not aware of our [Percy test setup](https://github.com/mozilla/foundation.mozilla.org/blob/master/cypress/integration/endpoint-tests.js#L118), it won't be obvious for them to tell why we are creating bannered campaign pages in two steps and not just use Faker for all.

Reply:
Good call!

Reply:
```suggestion
    title = 'Initial test Bannered Campaign with fixed title'
```